templates/api/clients/ruby/Gemfile.lock.tpl

```diff
@@ -1,27 +1,29 @@
 {{- $_ := stencil.ApplyTemplate "skipGrpcClient" "ruby" -}}
+{{ file.Static }}
 PATH
   remote: .
   specs:
-    {{ .Config.Name }}_client (1.63.0)
-      grpc (~> 1.38)
+    {{ .Config.Name }}_client (0.0.0)
```

Comment:
`0.0.0` is to make it clear that this doesn't coincide with anything.

Reply:
issue: I think in order for this to work, we'd need to have a post-template rendering hook to run `bundle install` and set the number correctly.

Reply:
Sure, but that's a valid version. `0.0.0` isn't :disappointed:

Reply:
Oh, I guess that's an invalid version for new services, sigh.